extension_distribution.md

```diff
@@ -293,3 +316,31 @@ different ways labels and extensions play well together:
   to have resources communicate with custom controllers).
   
 But I've punted on the specifics of how this should work.
+
+### Concurrency
+
+Extension repos are installed under `~/.tilt-dev`.
```

Comment:
:wave: Would it be possible to be compliant with the XDG Base Directory Specification? [[specs](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) | [arch wiki](https://wiki.archlinux.org/title/XDG_Base_Directory)]

```suggestion
Extension repos are installed under `$XDG_DATA_HOME/tilt-dev` (default to `~/.local/share/tilt-dev`).
```

Reply:
i like that idea! I've been meaning to move ~/.tilt-dev stuff to a more conventional place. also filed https://github.com/tilt-dev/tilt/issues/4850

Reply:
I was not aware Tilt was already using `~/.tilt-dev`, my bad: I did not mean to get off-topic :slightly_smiling_face: Thanks for the issue anyway!